types/options.d.ts

```diff
@@ -80,7 +80,7 @@ export interface ComponentOptions<
   updated?(): void;
   activated?(): void;
   deactivated?(): void;
-  errorCaptured?(): boolean | void;
+  errorCaptured?(err: Error, vm: V, info: string): boolean | void;
```

Comment:
> vm the component instance that triggered the error

I think the type should be `Vue` because error can be propagated.

Reply:
Indeed, thanks for pointing this out!